### PR TITLE
Feature/rename object names

### PR DESCRIPTION
**Pull request by michelu89**

### CODE REVIEW

File: core/libs/cache/src/lib/namespaces-cache.service.ts [hunk 1]
```diff
@@ -23,7 +23,9 @@ export class NamespacesCacheService {
   private namespaces = new Map<string, Map<string, CachedFile>>();
   private currentCachedFile: CachedFile;
 
-  constructor(public sidebarService: SidebarService) {}
+  constructor(public sidebarService: SidebarService) {
+    window['cache'] = this;
```
Comment (georgemoscu) [inline core/libs/cache/src/lib/namespaces-cache.service.ts]: This should be removed

File: core/libs/mx-graph/src/lib/services/mx-graph-geometry-provider.service.ts [hunk 2]
```diff
@@ -62,6 +62,8 @@ export class MxGraphGeometryProviderService {
         return new mxGeometry(x, y, ExpandedEllipseShape.expandedEllipseElementWidth, ExpandedEllipseShape.expandedEllipseElementHeight);
       case this.isRoundedBorderShape(metaModelElement):
         return new mxGeometry(x, y, ExpandedRoundBorderShape.expandedRoundBorderWidth, ExpandedRoundBorderShape.expandedRoundBorderHeight);
+      case this.isMiniShape(metaModelElement):
```
Comment (georgemoscu) [inline core/libs/mx-graph/src/lib/services/mx-graph-geometry-provider.service.ts]: This was removed a few weeks ago. Why do you need it?

File: core/libs/mx-graph/src/lib/services/mx-graph-geometry-provider.service.ts [hunk 3]
```diff
@@ -84,6 +86,8 @@ export class MxGraphGeometryProviderService {
           ExpandedRoundBorderShape.collapsedRoundBorderWidth,
           ExpandedRoundBorderShape.collapsedRoundBorderHeight
         );
+      case this.isMiniShape(metaModelElement):
```
Comment (georgemoscu) [inline core/libs/mx-graph/src/lib/services/mx-graph-geometry-provider.service.ts]: Same here?

**Review by georgemoscu — CHANGES_REQUESTED** [review]

File: core/libs/cache/src/lib/namespaces-cache.service.ts (hunk 1 above)
Comment (michelu89) [reply core/libs/cache/src/lib/namespaces-cache.service.ts]: I will check this

File: core/libs/mx-graph/src/lib/services/mx-graph-geometry-provider.service.ts (hunk 2 above)
Comment (michelu89) [reply core/libs/mx-graph/src/lib/services/mx-graph-geometry-provider.service.ts]: I will check this

File: core/libs/mx-graph/src/lib/services/mx-graph-geometry-provider.service.ts (hunk 3 above)
Comment (michelu89) [reply core/libs/mx-graph/src/lib/services/mx-graph-geometry-provider.service.ts]: I will check this

**Review by georgemoscu — APPROVED** [review]

**Pull request merged**